frontend/src/components/LevelMissionInfoBanner/LevelMissionInfoBanner.test.tsx

```diff
@@ -41,6 +43,8 @@ describe('LevelMissionInfoBanner component tests', () => {
 			<LevelMissionInfoBanner
 				currentLevel={currentLevel}
 				openOverlay={openOverlayMock}
+				openLevelsCompleteOverlay={() => {}}
```

Comment:
I suppose for the sake of completion, you could test that the openLevelsCompleteOverlay callback is fired on click, similar to the openOverlayMock. But there's not much to gain from it to be honest, there are bigger fish to fry.

Reply:
it'll take two seconds to put in the test no problem